src/data/quality-assurance-quiz.ts

```diff
@@ -1,15 +1,57 @@
 const qualityAssuranceQuiz = [
-    {
-        Question: "What does QA and QC stand for?",
-        Answer: "Quality Assurance and Quality Control ",
-        Distractor1: "Quality Assets and Quality completion",
-        Distractor2: "Quality Analyst and Quantum Computing",
-        Distractor3: "Quality Assessment and Quality Computing",
-        Explanation:
-          "Quality Assurance (commonly known as QA) and Queuing Control (commonly known as QC)",
-        Link: "https://www.freecodecamp.org/news/software-quality-assurance-guide/",
-      },
-    
+  {
+    Question: "What does QA and QC stand for?",
+    Answer: "Quality Assurance and Quality Control ",
+    Distractor1: "Quality Assets and Quality completion",
+    Distractor2: "Quality Analyst and Quantum Computing",
+    Distractor3: "Quality Assessment and Quality Computing",
+    Explanation:
+      "Quality Assurance (commonly known as QA) and Queuing Control (commonly known as QC)",
+    Link: "https://www.freecodecamp.org/news/software-quality-assurance-guide/",
+  },
+  {
+    Question: "What type of test involves catching errors while writing code?",
+    Answer: "Static Analysis",
+    Distractor1: "Unit",
+    Distractor2: "Integration",
+    Distractor3: "End to End",
+    Explanation:
+      "One of the most common sources of bugs is related to typos and incorrect types. Static Analysis Testing lets you catch typos and type errors as you write the code.",
+    Link: "https://kentcdodds.com/blog/static-vs-unit-vs-integration-vs-e2e-tests",
+  },
+  {
+    Question:
+      "What type of test involves testing individual functions or components?",
+    Answer: "Unit",
+    Distractor1: "Static Analysis",
+    Distractor2: "Integration",
+    Distractor3: "End to End",
+    Explanation:
+      "Unit Testing lets us verify that individual, isolated parts work as expected.",
+    Link: "https://kentcdodds.com/blog/static-vs-unit-vs-integration-vs-e2e-tests",
```

Comment:
While Ken't articles are very good, the team does want to stick with freeCodeCamp resources like articles or youtube videos.  The goal is to always promote freeCodeCamp resources when possible. 

https://www.freecodecamp.org/news/unit-tests-explained/